engine/app/helpers/good_job/application_helper.rb

```diff
@@ -1,5 +1,9 @@
 # frozen_string_literal: true
 module GoodJob
   module ApplicationHelper
+    def relative_time(timestamp)
+      text = timestamp.future? ? "in #{time_ago_in_words(timestamp)}" : "#{time_ago_in_words(timestamp)} ago"
+      content_tag(:time, text, datetime: timestamp, title: timestamp)
```

Comment:
I think `content_tag` is deprecated in favor of `tag.time`.

Reply:
I changed it. I wasn't sure in which version of Rails `tag` was introduced, but we're already using it in engine, so it is fine.